prog/dftb+/lib_common/unitconversion.F90

```diff
@@ -205,12 +201,45 @@ module dftbp_unitconversion
       &/)
 
 
+  !> Number of mass units
+  integer, parameter :: nMassUnit = 6
+
+
   !> Mass units
   type(unit), parameter :: MassUnits(nMassUnit) = (/ &
       &unit("au                  ", 1.0_dp), &
       &unit("amu                 ", amu__au ), &
       &unit("da                  ", amu__au ), &
-      &unit("dalton              ", amu__au ) &
+      &unit("dalton              ", amu__au ), &
+      &unit("kg                  ", kg__au ), &
+      &unit("g                   ", 1.0e+3_dp*kg__au ) &
       &/)
 
+
+  !> Number of mass density units
+  integer, parameter :: nMassDensityUnit = 19
+
+
+  type(unit), parameter :: massDensityUnits(nMassDensityUnit) = [ &
```

Comment:
Having a total of 19 units for the mass density is certainly overkill, but covers the ones I actually need + all the combinations from mass and volume.